### DISCUSSION TITLE
How to change transition?

### DISCUSSION THREAD

**Discussion by JenuelDev** [vue-final/vue-final-modal#297] [Q&A] [answered]
How do I change the transition when it goes in and goes out?

**Comment by hunterliu1003** [comment, marked as answer]
You can use contentTransition or overlayTransition props to customize the transition. vue-final-modal provide a few built-in transition, you can also pass Vue TransitionProps to create your own.

Check more detail in https://vue-final-modal.org/api/components/vue-final-modal/#contenttransition
